pkg/apis/flowcontrol/types.go

```diff
@@ -223,17 +234,25 @@ type PriorityLevelConfigurationSpec struct {
 	// `queues` is a number of queues that belong to a non-exempt PriorityLevelConfiguration object. The queues exist
 	// independently at each apiserver. The value must be positive for a non-exempt priority level and setting it to 1
 	// disables shufflesharding and makes the distinguisher method irrelevant.
+	// TODO: sugguest a default or a way of deciding on a value.
 	// +optional
 	Queues int32
 	// `handSize` is a small positive number for applying shuffle sharding. When a request arrives at an apiserver the
 	// request flow identifier’s string pair is hashed and the hash value is used to shuffle the queue indices and deal
 	// a hand of the size specified here. If empty, the hand size will the be set to 1.
+	// NOTE: To figure out a better value for your cluster, please refer to (#76846)[https://github.com/kubernetes/kubernetes/issues/76846#issuecomment-523700960]
```

Comment:
That is good enough for now.

The referenced information should go in doc and/or blog, and the reference to it should clearly say that this is about the choice for both number of queues and hand size.

To me, "To figure out a better value" is a bit awkward because it presume that there is already some value figured out.  I think our goal is to help admins make a good choice in the first place.  So I would word it something more like "To figure out a good value".

The comment on this field should point out that it is relevant only if `Exempt` is false.